### DISCUSSION TITLE
notes/aaroh-mankad/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#44] [Announcements]
# notes/aaroh-mankad/

Notes about the cascadiajs 2022 conference

https://cascadiajs.dendron.so/notes/aaroh-mankad/

**Comment by vim-usds** [comment]
Slides here - https://docs.google.com/presentation/d/1RXbLxBHNUR7cqgF6_uxBJdE9A2qNO8Cu5n_tNscfV6M/edit#slide=id.g10386427126_2_12021
